### ISSUE TITLE
Not getting an output file (True Prediction dataset size 0)

### ISSUE THREAD

**Issue by Omer-Sella** [ZKI-PH-ImageAnalysis/seq2squiggle#13]
I'm trying to use the simulator with the provided example fasta files and getting no output file. The log shows prediction data set size to be 0, but this is consistent over other fasta files, as well as other execution options.
Any thoughts ?

$ seq2squiggle predict test.fasta -o ./example.blow5 -n 10000
seq2squiggle INFO 12:18:27: seq2squiggle version 0.3.1
seq2squiggle INFO 12:18:27: Arguments:
seq2squiggle INFO 12:18:27:  fasta: test.fasta
seq2squiggle INFO 12:18:27:  read_input: False
seq2squiggle INFO 12:18:27:  num_reads: 10000
seq2squiggle INFO 12:18:27:  read_length: 10000
seq2squiggle INFO 12:18:27:  coverage: -1
seq2squiggle INFO 12:18:27:  out: example.blow5
seq2squiggle INFO 12:18:27:  profile: dna-r10-prom
seq2squiggle INFO 12:18:27:  noise_sampler: True
seq2squiggle INFO 12:18:27:  duration_sampler: True
seq2squiggle INFO 12:18:27:  dwell_mean: 9.0
seq2squiggle INFO 12:18:27:  dwell_std: 0.0
seq2squiggle INFO 12:18:27:  noise_std: 1.0
seq2squiggle INFO 12:18:27:  distr: expon
seq2squiggle INFO 12:18:27:  predict_batch_size: 1024
seq2squiggle INFO 12:18:27:  export_every_n_samples: 1000000
seq2squiggle INFO 12:18:27:  sample_rate: 5000
seq2squiggle INFO 12:18:27:  digitisation: None
seq2squiggle INFO 12:18:27:  range: None
seq2squiggle INFO 12:18:27:  offset_mean: None
seq2squiggle INFO 12:18:27:  offset_std: None
seq2squiggle INFO 12:18:27:  median_before_mean: None
seq2squiggle INFO 12:18:27:  median_before_std: None
seq2squiggle INFO 12:18:27:  seed: 385
seq2squiggle INFO 12:18:27:  model: None
seq2squiggle INFO 12:18:27:  config: None
seq2squiggle INFO 12:18:27:  verbosity: info
seq2squiggle INFO 12:18:27: Config file was not specified. Default config will be used.
seq2squiggle INFO 12:18:27: Setting seeds using random seed 385
seq2squiggle INFO 12:18:27: Weights file path is not provided.
seq2squiggle INFO 12:18:27: Detected R10.4.1 chemistry profile.
seq2squiggle INFO 12:18:27: Profile can be changed with the --profile parameter
seq2squiggle INFO 12:18:27: Found matching weights in local cache: .cache/seq2squiggle/R10_4_1@v0.3.0.ckpt
seq2squiggle INFO 12:18:28: Genome mode. Reads will be generated from input fasta: test.fasta
seq2squiggle INFO 12:19:20: True Prediction dataset size 0
py.warnings WARNING 12:19:20: /pyenv/versions/3.10.0/envs/python310/lib/python3.10/site-packages/pytorch_lightning/utilities/data.py:106: Total length of `DataLoader` across ranks is zero. Please make sure this was your intention.

py.warnings WARNING 12:19:20: /pyenv/versions/3.10.0/envs/python310/lib/python3.10/site-packages/pytorch_lightning/utilities/data.py:106: Total length of `DataLoader` across ranks is zero. Please make sure this was your intention.

seq2squiggle INFO 12:19:20: Prediction done.

**Comment by Omer-Sella** [comment]
(Eewew) Thank you, yes it seems to work now ! So am I correct in thinking that running:
$ seq2squiggle predict ./someDNA.fasta -o example.blow5 --read-input
Will produce 1 squiggle ( == one output from one pore ?) corresponding to each sequence in someDNA.fasta ?

**Comment by denisbeslic** [comment]
Yes, correct: Each complete sequence will be simulated as a signal.

**Comment by Omer-Sella** [comment]
Thank you, yes it works,
